### DISCUSSION TITLE
glb export/import

### DISCUSSION THREAD

**Discussion by vegatss** [atteneder/glTFast#372] [Q&A] [answered]
I can export a game object as glb at runtime. Everything goes smoothly. But when I try to import the same glb I get this error: ArgumentException: JSON parse error: Missing a name for object member.

Also when I try to open the glb I exported it doesn't open. Is there a problem with the exporting
![Ekran Alıntısı](https://user-images.githubusercontent.com/30438433/159268165-7d8bccd8-364f-417a-8d61-7d90cf4f3d99.PNG)
?

**Comment by Daniel4144** [comment]
#374 should be the same problem - as a workaround you could try to use a different shader for the exported objects

**Reply by atteneder** [reply]
True, this is likely related. Please see, I just commented and closed it.

**Comment by atteneder** [comment, marked as answer]
I think this is a bug that's already fixed in the `main` branch and upcoming release. In such case, please create an issue instead of a discussion.

Thanks!
